### PR TITLE
Run/process card for 2HDM HPlus + h production

### PR DESCRIPTION
**Pull request by DaveBrun94**
These are the run and process cards for a 2HDM charged Higgs production in the association with a light CP even Higgs decaying to a lepton + four b-jet final state. It will be produced for UL.

It will be produced in the a grid of charged Higgs mass and light Higgs mass. My theorist colleagues provided SLHA files for each point in the grid, which are used as parameter cards. Should also these files be commited?

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/HPlusAndH_ToWHH_ToL4B/HPlusAndH_ToWHH_ToL4B_run_card.dat [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+ F        = fixed_ren_scale  ! if .true. use fixed ren scale
+ F        = fixed_fac_scale  ! if .true. use fixed fac scale
+ 91.1880  = scale            ! fixed ren scale
+ 91.1880  = dsqrt_q2fact1    ! fixed fact scale for pdf1
+ 91.1880  = dsqrt_q2fact2    ! fixed fact scale for pdf2
+ 1        = scalefact        ! scale factor for event-by-event scales
+#*********************************************************************
+# Matching - Warning! ickkw > 1 is still beta
+#*********************************************************************
+ 0        = ickkw            ! 0 no matching, 1 MLM, 2 CKKW matching
```
Comment (agrohsje) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/HPlusAndH_ToWHH_ToL4B/HPlusAndH_ToWHH_ToL4B_run_card.dat:66]: you have additional jets and should apply mlm.

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/HPlusAndH_ToWHH_ToL4B/HPlusAndH_ToWHH_ToL4B_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+ 0.4  =  dparameter 
+#*********************************************************************
+# maximal pdg code for quark to be considered as a light jet         *
+# (otherwise b cuts are applied)                                     *
+#*********************************************************************
+ 4 = maxjetflavor    ! Maximum jet pdg code
+#*********************************************************************
+# Jet measure cuts                                                   *
+#*********************************************************************
+ 0   = xqcut   ! minimum kt jet measure between partons
```
Comment (agrohsje) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/HPlusAndH_ToWHH_ToL4B/HPlusAndH_ToWHH_ToL4B_run_card.dat:260]: needs adjustement for mlm.

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/HPlusAndH_ToWHH_ToL4B/HPlusAndH_ToWHH_ToL4B_run_card.dat [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+#*********************************************************************
+# KT DURHAM CUT                                                      *
+#*********************************************************************
+ -1    =  ktdurham        
+ 0.4  =  dparameter 
+#*********************************************************************
+# maximal pdg code for quark to be considered as a light jet         *
+# (otherwise b cuts are applied)                                     *
+#*********************************************************************
+ 4 = maxjetflavor    ! Maximum jet pdg code
```
Comment (agrohsje) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/HPlusAndH_ToWHH_ToL4B/HPlusAndH_ToWHH_ToL4B_run_card.dat:256]: is the model 4FS or 5FS per default?

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/HPlusAndH_ToWHH_ToL4B/HPlusAndH_ToWHH_ToL4B_proc_card.dat [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import model_v4 2HDMC
+define hc = h- h+
+define w = w+ w-
+define l = l+ l-
+define v = vl vl~
+generate p p > hc h1, h1 > b b~, (hc > h1 w, h1 > b b~, w > l v)
```
Comment (agrohsje) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/HPlusAndH_ToWHH_ToL4B/HPlusAndH_ToWHH_ToL4B_proc_card.dat:6]: You don't want to include W -> tau decays, where the tau decays in P8 in e or mu (or jet)?

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/HPlusAndH_ToWHH_ToL4B/HPlusAndH_ToWHH_ToL4B_run_card.dat (hunk 3 above)
Comment (DaveBrun94) [reply bin/MadGraph5_aMCatNLO/cards/production/13TeV/HPlusAndH_ToWHH_ToL4B/HPlusAndH_ToWHH_ToL4B_run_card.dat:256]: The model is per default in the 4FS, massive b-quarks are used

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/HPlusAndH_ToWHH_ToL4B/HPlusAndH_ToWHH_ToL4B_proc_card.dat (hunk 4 above)
Comment (DaveBrun94) [reply bin/MadGraph5_aMCatNLO/cards/production/13TeV/HPlusAndH_ToWHH_ToL4B/HPlusAndH_ToWHH_ToL4B_proc_card.dat:6]: You are right, I will include them

**Comment by DaveBrun94** [comment]
Thanks for your comments. I activated the MLM matching and added in the tau leptons in the lepton definition in my run card

**Comment by agrohsje** [comment]
Looks good to me. Thanks. @GurpreetSinghChahal  please check and merge.

**Pull request merged**